src/style.css

```
@@ -0,0 +1,147 @@
+html,
+body,
+div,
+a,
+ul,
+p,
+li,
+header,
+nav {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```

move box-sizing to * selector
redundant to reset margin and padding for all tags which exist in project
some doesn't have them by default
```suggestion
body,
ul,
p {
  margin: 0;
  padding: 0;
```

---

Done